lib/generators/flipper/templates/migration.rb

```diff
@@ -12,12 +12,10 @@ def self.up
       t.string :value
       t.timestamps null: false
     end
-    add_foreign_key :flipper_gates, :flipper_features, on_delete: :cascade
```

Comment:
Reviewed the code and it doesn't look like I ever did need to touch these - they were working out of the box. What were they breaking for you?